src/init.cpp

```diff
@@ -1106,6 +1113,9 @@ bool AppInit2(ThreadHandlerPtr threads)
         if (file) {
             filesystem::path pathBootstrapOld = GetDataDir() / "bootstrap.dat.old";
             LoadExternalBlockFile(file);
+
+            UpdateOutOfSyncByAge();
+
```

Comment:
Same here.

Reply:
Agreed. The only place we need them is the block handlers in main...